tern/formats/default/generator.py

```diff
@@ -101,7 +101,8 @@ def get_layer_info_list(layer):
     layer_file_licenses_list = []
     file_level_licenses = None
     package_list = PrettyTable()
-    package_list.field_names = ["Package", "Version", "License", "Pkg Format"]
+    package_list.field_names = ["Package", "Version", "License", "Pkg Format",
+                                "Pkg Licenses"]
```

Comment:
Hmm, I'm not sure we should add a separate `pkg_licenses` column. I suggest changing the existing `License` to `License(s)`.

Reply:
@rnjudge makes sense. Just updated the PR.